### DISCUSSION TITLE
Major Enhancements

### DISCUSSION THREAD

**Discussion by IceWaffles** [EssentialsX/Essentials#4624] [Ideas and Feature Suggestions] [answered]
Allow viewing of offline players' inventories and enderchests. Also, allow access to armour and offhand during /invsee (player). Teleporting to a offline players' bed would also be useful if possible. These are some major annoyances that I run into very often while using essentials, and it would make everything so much easier if these things could be added.

**Comment by JRoy** [comment, marked as answer]
This is out of the scope of Essentials as it would involve raw data parsing. Please use a dedicated plugin like OpenInv for this.
